packages/loot-core/src/platform/server/fs/index.web.ts

```diff
@@ -77,15 +78,15 @@ async function _readFile(filepath, opts?: { encoding?: string }) {
     !filepath.endsWith('.sqlite')
   ) {
     if (!_exists(filepath)) {
-      throw new Error('File does not exist: ' + filepath);
+      throw new Error(t('File does not exist:') + ' ' + filepath);
     }
 
     // Grab contents from IDB
     const { store } = idb.getStore(await idb.getDatabase(), 'files');
     const item = await idb.get(store, filepath);
 
     if (item == null) {
-      throw new Error('File does not exist: ' + filepath);
+      throw new Error(t('File does not exist:') + ' ' + filepath);
```

Comment:
Here too - I wont put a comment on all of them but it would be ideal to remove them all 👍